src/main/java/org/osc/manager/rest/server/SecureManagerServerRestConstants.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.rest.server;
+
+public interface SecureManagerServerRestConstants {
+    // Because we started with versions in the URL, we will continue to have v1 in the URL
+    String SERVER_API_PATH_PREFIX = "/sm/server/v1";
```

Comment:
Since you have /sm/ here change the url from /testsmplugin/sm/.... to /testplugin/sm/ and for the sdn controller it will be /testplugin/sdn/....